_templates/layout.html

```diff
@@ -0,0 +1,41 @@
+{%- extends "alabaster/layout.html" %}
```

Comment:
This doesn't work for me (`TemplateNotFound: alabaster/layout.html`) - but the following does work:

```
{%- extends "!layout.html" %}
```

http://sphinx-doc.org/templating.html